torch/nn/modules/normalization.py

```diff
@@ -68,6 +72,7 @@ def extra_repr(self):
         return '{size}, alpha={alpha}, beta={beta}, k={k}'.format(**self.__dict__)
 
 
+@weak_module
 class LayerNorm(Module):
```

Comment:
Where is this getting tested ?

Reply:
There's 4 `LayerNorm` tests in `test_nn.py` that get run for this